### ISSUE TITLE
How to define a new loss function in CNTK?

### ISSUE THREAD

**Issue by jingyonghou** [Microsoft/CNTK#892]
How to define a new loss function in CNTK? Is there any tutorials？

**Comment by dongyu888** [comment]
If the loss function can be computed using existing operations you can just implement it using brain script. If this is not possible than you can choose CrossEntropyWithSoftmax as an example to develop your own. Search the whole project for CrossEntropyWithSoftmaxNode and you will know what to change. It’s relatively straight forward.

**Comment by jingyonghou** [comment]
Thanks for your reply.
P.S. I like your book "Automatic Speech Recognition – A Deep Learning Approach"

**Comment by dongyu888** [comment]
glad that you like my book. closing this issue.
